community/urls.py

```diff
@@ -104,10 +98,10 @@ def get_organization():
         distill_file='twitter/index.html',
     ),
     distill_url(
-        r'log/', log_index,
-        name='log',
+        r'CI/Build/', BuildLogsView.as_view(),
```

Comment:
`r'ci/build'`

Why capital?

Reply:
As CI refers to Continuos Integration, so I was going with caps Initials. If not OK, I will relative url lowercase